### ISSUE TITLE
Claymore style explosives

### ISSUE THREAD

**Issue by kevingranade** [CleverRaven/Cataclysm-DDA#28766]
### Expected behavior
Claymore style explosives would project fragments in a designated area and cause minimal damage elsewhere.

### Actual behavior
These explosives dont exist.

### Details
These kinds of explosives are particularly useful tactically as they can cause a huge number of casualties in a designated area with minimal risk to the user.
This will require minimal code changes as long as the arc of projectiles fall on 45deg increments.

**Comment by thethunderhawk** [comment]
Could this be adapted for breaching charges as well? That’s a real wishlist item for me and it sounds mechanically very similar.

**Comment by Shodan14** [comment]
Could this code be later turned around for shaped charges for blowing holes into walls etc.?

**Comment by Zombiemanpig** [comment]
Could this be added to ammo too? I want to mod in claymore arrows.

**Comment by kevingranade** [comment]
If you want a different feature, open an issue about it.

**Comment by Cenness** [comment]
I stumbled upon this issue recently - https://github.com/CleverRaven/Cataclysm-DDA/issues/50504 - couldn't it be used to implement directed explosions?

**Comment by kevingranade** [comment]
No, explosions use raytracing code to decide what tiles are potentially hit by fragments, the spell system uses a cheaper and easier to work with line drawing or flood fill thing that's a lot more approximate.  I'd actually turn it around and say that the directional area spells should switch to using the raytracing system.
